lib/modules/versioning/swift/range.ts

```diff
@@ -6,6 +6,7 @@ const fromParam = regEx(/^\s*from\s*:\s*"([^"]+)"\s*$/);
 const fromRange = regEx(/^\s*"([^"]+)"\s*\.\.\.\s*$/);
 const binaryRange = regEx(/^\s*"([^"]+)"\s*(\.\.[.<])\s*"([^"]+)"\s*$/);
 const toRange = regEx(/^\s*(\.\.[.<])\s*"([^"]+)"\s*$/);
+const vPrefix = regEx(/v([0-9]+)/);
```

Comment:
```suggestion
const vPrefix = regEx(/^v([0-9]+)/);
```
should only be at start of version value